src/main.ts

```diff
@@ -138,7 +138,16 @@ export async function run(): Promise<void> {
       ...(vertexApiKey ? {VERTEX_API_KEY: vertexApiKey} : {}),
       ...(cachePath ? {PROMPTFOO_CACHE_PATH: cachePath} : {}),
     };
-    await exec.exec(`npx promptfoo@${version}`, promptfooArgs, {env});
+    try {
+      await exec.exec(`npx promptfoo@${version}`, promptfooArgs, {env});
+    } catch (error) {
+      // Ignore nonzero exit code
+      if (error instanceof Error) {
+        core.info(`promptfoo exited with error: ${error.message}`);
+      } else {
+        core.info(`promptfoo exited with an unknown error.`);
+      }
+    }
```

Comment:
I think it's best to post the result of a failed run and still throw error (fail the CI job run)

Reply:
That way, we don't have a job run where there's a failed assertion and the CI passes.

Reply:
If it does not align with your philosophy is here, you should consider exposing a flag/option for this.